static/css/eval_retrieval_generation.scss

```diff
@@ -188,13 +189,12 @@ label {
 
   #form-container {
     border-top: solid 1px var(--gm-3-sys-light-outline-variant);
-    gap: 16px;
```

Comment:
curious why you're removing this, how do the other feedback panes look with this change?

Reply:
looks like this is block by some other css.